roachtest/sequelize: add test for sequelize

otan opened the pull request:
This adds tests for cockroachdb-sequelize. Running on the latest version
that's published; only 19 tests so far. I'll leave to to someone else to
bump.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/63272)
<!-- Reviewable:end -->

rafiss commented on pkg/cmd/roachtest/sequelize.go:
```diff
@@ -0,0 +1,154 @@
+		rawResults, err := c.RunWithBuffer(ctx, t.l, node,
+			`cd /mnt/data1/sequelize/ && npm test`,
+		)
+		rawResultsStr := string(rawResults)
```
i think we'd want to have the blocklists/ignorelists in place too. otherwise we wouldn't know if it fails

or if no failures are expected, then the roachtest should just automatically fail if any tests fail and point people at the logs

RichardJCai replied:
Yeah I'm gonna add that after, for some reason I had this test working but now I'm running into a certs issue :/

otan replied:
> i think we'd want to have the blocklists/ignorelists in place too. otherwise we wouldn't know if it fails

isn't that what the `err != nil` check does below? `RunWithBuffer` returns an error if exit error code != 0

rafiss replied:
oh whoops yes!

rafiss reviewed (approved):
needs a bazel-generate but looks good!

otan committed:
roachtest/sequelize: add test for sequelize

This adds tests for cockroachdb-sequelize. Running on the latest version
that's published; only 19 tests so far. I'll leave to to someone else to
bump.

Release note: None

otan commented:
bors r=rafiss
